### ISSUE TITLE
Data types, pixels and error message

### ISSUE THREAD

**Issue by Hypnootika** [qb-0/pyMeow#33]
Hey, first of all, thanks a lot for this project. Ive been searching for something like this (that combines all the things i need). 

While i worked myself in, i stumbled about 1 or 2 things,  so i thought, I'd share my experience and report some "not really convenient things"

Data types und error message are basically one thing. 

I was trying to feed the new_color function an instance of my color class. And i received the error:

Python types arent compatible (free out of my mind).
That made me wonder so i went ahead and checked if you provide own data types.

Long story short:

Problem was i used a float representation for the color 
1.0, 1.0, 1.0, 1.0
So it would be nice if you could adjust either the error message or maybe own data types?

As for the pixels(this is just a kind request):

It would be awesome if you could extent the base array usage with an appropriate single Pixel detection method, that doesnt take a color but rather x, y

Thanks again

**Comment by qb-0** [comment]
Hey,

thank you for the feedback and your issue.

The exception messages are caused by nimpy which is the python bridge from nim to python. So it's not really on pyMeows end. However it might be a good point that there should be a color function which accepts float values as well.

Regarding your pixel scan issue: I'm not sure if I got you but wouldn't `list(pixel_enum_region(x, y, 1, 1))` do it?
